packages/connect-wallet-modal/src/components/Ledger/LedgerConnectionScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Stack, StackItem, Text, useBreakpoint } from '@lidofinance/lido-ui';
+import { LedgerImageDefault } from './icons/LedgerImageDefault';
+import { LedgerImageDefaultMobile } from './icons/LedgerImageDefaultMobile';
+import { LedgerScreenContainerStyled } from './styles';
+
+export const LedgerConnectionScreen = () => (
+  <LedgerScreenContainerStyled>
+    <Stack direction="column" spacing="xl" align="center">
+      <StackItem>
+        {useBreakpoint('md') ? (
```

Comment:
someone else would say it is bad to write like that but I personally love using hooks inside jsx :)